src/main/java/org/example/homework5/RandomNumberSum.java

```diff
@@ -0,0 +1,28 @@
+package org.example.homework5;
+import java.util.Random;
+
+public class RandomNumberSum {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] occurrences = new int[121111111];
```

Comment:
It is better to create
`int[] occurrences = new int[101];`

and then iside the loop check `if (occurrences[randomNumber] == 2)` -> exit the loop